testdata/p4_16_samples_outputs/issue1937-1-bmv2-frontend.p4

```diff
@@ -0,0 +1,60 @@
+#include <core.p4>
+#include <v1model.p4>
+
+header h1_t {
+    bit<8> f1;
+    bit<8> f2;
+}
+
+struct headers_t {
+    h1_t h1;
+}
+
+struct metadata_t {
+}
+
+control ingressImpl(inout headers_t hdr, inout metadata_t meta, inout standard_metadata_t stdmeta) {
+    @name(".foo") action foo(out bit<8> x, in bit<8> y=5) {
+        x = y >> 2;
+    }
+    @name(".foo") action foo_0(out bit<8> x_1, in bit<8> y_1=5) {
+        x_1 = y_1 >> 2;
+    }
+    bit<8> tmp;
+    bit<8> tmp_0;
+    apply {
+        tmp_0 = hdr.h1.f1;
+        foo(tmp, tmp_0);
+        hdr.h1.f1 = tmp;
+        foo_0(x = hdr.h1.f2, y = 8w5);
```

Comment:
I do not see how this intermediate code can compile, given that action foo_0 above has parameters named x_1 and y_1, not x and y.

Reply:
This was a bug in the code that emitted P4, internally the names were correct. That's why the programs would compile correctly. I will add a commit to fix this in this PR.